manual/docker/Enterprise-Edition/Deploy SeaTable-EE with Docker.md

```diff
@@ -97,6 +94,16 @@ docker-compose up -d
 
 NOTE: You should run the above command in the directory with the `docker-compose.yml`.
 
+
+### Activating the SeaTable License
+
+Save the license file in the directory `/opt/seatable/seatable-data/seatable`. Make sure that the name is seatable-license.txt.
```

Comment:
I'd expect to see "seatable-license.txt" as `seatable-license.txt` (backtick-quotes).

Reply:
Modification made